diktat-rules/src/main/kotlin/com/saveourtool/diktat/ruleset/rules/chapter1/IdentifierNaming.kt

```diff
@@ -412,7 +412,9 @@ class IdentifierNaming(configRules: List<RulesConfig>) : DiktatRule(
             @Suppress("COLLAPSE_IF_STATEMENTS")
             if (functionReturnType != null && functionReturnType == PrimitiveType.BOOLEAN.typeName.asString()) {
                 @Suppress("COLLAPSE_IF_STATEMENTS")
-                if (allMethodPrefixes.none { functionName.text.startsWith(it) }) {
+                val isOperatorFun = node.firstChildNode?.findChildrenMatching { it.elementType == KtNodeTypes.OPERATION_REFERENCE }
```

Comment:
I'd move this to a separate extension function of `ASTNode` and put the function invocation into the higher `if` so that it would become 
```kotlin
if(node.hasBooleanReturnType() && !node.isOperatorFun()) { ... }
```